package.json

```diff
@@ -128,10 +124,7 @@
     "babel-preset-gatsby": "^0.5.8",
     "eslint": "^8.38.0",
     "eslint-config-sentry-docs": "^1.114.0",
-    "est": "^0.2.1-alpha",
-    "gh-pages": "^2.2.0",
```

Comment:
unused, probably left over from a time where we tried deploying this to gh pages before vercel